libs/custom-paper-handler/src/driver/driver_interface.ts

```diff
@@ -0,0 +1,112 @@
+import { Coder, CoderError, Uint16, Uint8 } from '@votingworks/message-coder';
```

Comment:
I added this interface since we'll need one for an eventual hardware mock for dev environments.